Makefile

```diff
@@ -0,0 +1,54 @@
+####### Universal ######
+
+pip-publish:
+	pip install -r requirements.txt
+	pip install -e .
+
+pip-test:
+	pip install -r requirements.txt
+	pip install .
+
+unit-test:
+	pytest -sv tests/unit
+
+blackbox-smoke-prefix:
+	echo "hello blackbox!"
+	pwd
+	ls -l
+	ls -l sandbox
+	cd sandbox && docker-compose ps
+
+integration-test:
+	pytest -sv tests/integration
+
+
+###### Mac Only ######
```

Comment:
@tzaffi Asking having briefly reviewed the PR:  What is Mac-specific about these build commands?

Though I'm unfamiliar with some of the tooling (e.g. tox), at first glance, I'm _guessing_ it's label's _Mac only_ because that's the only environment tested locally.

My default assumption is that the build is _not_ OS-specific.

Reply:
My original intent with the `mac-*` prefix was to differentiate between commands that I know will be successful on github, and ones that aren't. Also, by using `mac` I'm hinting at the fact that I only know really that these can work on our Algorand laptops and make no guarantee for other os'es. But the naming is a bit jarring. Do you think `local-*` would be better?

Reply:
@tzaffi Got it - Unless we know about OS-specific limitations, I'd opt for something like `local`.

I totally see your concern that we're not exercising the commands on multiple OSes, so we can't be _certain_.  But, I'm hoping we can cross that bridge should it arise rather than starting from the OS-specific assumption.

Ultimately, the discussion points us back towards unifying the local and CI builds.  If you feel strongly, I am OK to keep as is.  There's bigger fish to fry.

Reply:
@tzaffi Thanks for the naming change.